project/RELEASES.md

```diff
@@ -0,0 +1,172 @@
+        -- failure -->
+        debateReg[\"Impact of both new and old regression
+                    and possible workarounds
+                    are discussed by the compiler team."/]
+
+        regFixReg -- success --> ocbReg
+    end
+    ocbReg -- no --> acc
+    debateReg -->|"decision on
+                   the recommended
+                   workarounds for
+                   the new regression"| acc
+    debateReg -->|"decision on
+                   the recommended
+                   workarounds for
+                   the old regression"| reject
+
+    regression -- no --> types
+    regIsLTS -- no --> types
+    types{"Can the fix
+           change types
+           in any correct
+           Scala 3 code?"}
+    -- yes --> request
+    types -- no --> ocbOther
+
+    request{"Is backport
+             of the fix
+             heavily requested?"}
+    -- yes --> debateReq
+    request -- no --> reject
+
+    debateReq[\"Possibility of
+                the backport is discussed
+                by the compiler team"/]
+    --> |"backport is rejected"| reject
+    debateReq --> |"backport is accepted"| ocbOther
+
+    subgraph "Other Fixes"
+        ocbOther{"Does it cause
+                 any new failures
+                 in the full CB?"}
+
+        -- yes -->
+        regFixOther[\"Try to provide
+                    a followup fix for
+                    a regressions"/]
+        -- success --> ocbOther
+
+        ocbOther -- no -->
+        lint{"Does it introduce
+              any new warnings
+              behind flags?"}
+        -- yes -->
+        lintOcb{"Does it cause any
+                 new failures in the full CB
+                 after forcing a new flag?"}
+        -- yes --> regFixOther
+    end
+
+    lint -- no --> acc
+    lintOcb -- no --> acc
+    regFixOther -- failure --> reject
+
+    acc(["The PR is backported"])
+    reject(["The PR is not backported"])
+```
+
+CVE stands for Common Vulnerabilities and Exposures and in the chart above it means an issue with an CVE-ID assigned to it.
+
+CB stands for Community Build, and by full CB we mean the full run of [the Scala 3 Open Community Build](https://github.com/VirtusLab/community-build3).
+
+## How should things be backported?
+
+The backporting process is tracked by [a GitHub Project](https://github.com/orgs/lampepfl/projects/6) in the lampepfl organization. Every PR merged to the `main` branch is automatically added to the `Needs Assessment` column. Those PRs are reviewed by the release officer or other appointed person. They can decide to remove them from the project (backport rejected) or to move them to the `Backporting in progress` column. If the PR with the backport has any differences from the original PR, the person doing the backport will ask the author and reviewers of the original change to review the backport. After merging the backport, the PR will be moved to the `Backport done` column and, after the release, to the `Released` column.
```

Comment:
This link gives me a 404.

Maybe the project is private? If that's intended, I think it would still be nice to disclose that.

Reply:
I can open it in private browser window, so it shouldn't be private now.

Reply:
I can confirm that I do see it now 👍